src/trainers/ppo_trainer.py

```diff
@@ -1,28 +1,3 @@
-from functools import partial
+from .simple_pl_trainer import SimplePLTrainer
 
-import lightning.pytorch as pl
-from torch.utils.data import DataLoader
-
-from ..data_collectors.trajectory_data_collector import TrajectoryDataCollector
-from ..models.ppo_lit_module import PPOLitModule
-from .trainer import Trainer
-
-
-class PPOTrainer(Trainer):
-    def __init__(
-        self,
-        module: PPOLitModule,
-        data_collector: TrajectoryDataCollector,
-        dataloader: partial[DataLoader],
-        trainer: pl.Trainer,
-    ):
-        self.module = module
-        self.data_collector = data_collector
-        self.dataloader = dataloader
-        self.trainer = trainer
-
-    def train(self):
-        dataset = self.data_collector.get_data()
-        dataloader = self.dataloader(dataset=dataset)
-        self.trainer.fit(self.module, dataloader)
-        self.data_collector.clear()
+PPOTrainer = SimplePLTrainer
```

Comment:
PPOTrainerだけエイリアスではなく、実装の改変をお願いします。

```suggestion
class PPOTrainer(SimplePLTrainer):
    data_collector: DynamicsDataCollector
    def train(self):
       super().train()
       self.data_collector.clear()
```

Reply:
修正しました https://github.com/MLShukai/PrimitiveAMI/pull/110/commits/bec08433e922923f99a57e7ee3e633ace6032448